features/config.feature

```diff
@@ -92,6 +92,28 @@ Feature: Config
       The requested config file does not exist
       """
 
+  Scenario: Prioritize *.php config file instead of yaml
+    Given a file named "behat.php"
+    Given a file named "behat.yaml"
+    Given a some feature context
+    And a some feature scenarios
+    When I run behat in debug mode
+    Then the output should contain:
+      """
+      behat.php
+      """
```

Comment:
Prioritising behat.php over behat.yaml is a BC break. It's possible end-users already have a behat.php for some other purpose.

Reply:
I've moved the priorities.